src/main/java/net/minecraftforge/event/world/RaidEvent.java

```diff
@@ -0,0 +1,23 @@
+package net.minecraftforge.event.world;
+
+import net.minecraft.world.raid.Raid;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+/*
+ * This event is called right before a raid starts.
+ */
```

Comment:
1. Please use proper Javadoc formatting.
2. This documentation is not accurate. You call this event every tick while the raid is active. It would, imho. be more logical to have `RaidEvent.Tick` (not sure if cancelling that is even useful, does that mean "skip this tick" or "stop the raid"?).

Reply:
noted, added 4 subevents